### PR TITLE
Feature - TSS-770 - Change affected goods edit page

### PR DESCRIPTION
**Pull request by chris-pettinga**
Editing 'export types' on the barrier dashboard now takes you to the new edit pages which include checkboxes for each export type, and a description of the exported goods.

Renamed export_type to export_types in the BarrierExportTypeForm to align with the DB

Deleted references to the old journey as no longer required.

### CODE REVIEW

File: barriers/views/export_types.py [hunk 1]
```diff
@@ -50,67 +38,19 @@ def get_success_url(self):
...
+        initial["export_types"] = [each["name"] for each in self.barrier.export_types]
+        initial["export_description"] = self.barrier.export_description
+        return initial
+
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
-        kwargs["export_types"] = [(key, value) for key, value in EXPORT_TYPES]
         kwargs["barrier_id"] = str(self.kwargs.get("barrier_id"))
         kwargs["token"] = self.request.session.get("sso_token")
+        kwargs["token"] = self.request.session.get("sso_token")
```
Comment (osimuka) [inline barriers/views/export_types.py:51]: line already defined above

File: templates/reports/barrier_details_summary_wizard_step.html [hunk 2]
```diff
@@ -92,7 +92,7 @@ <h4 class="govuk-heading-s question-label">Name of companies affected by the bar
                 <h2 class="govuk-heading-m">Goods, services or investments affected</h2>
                 <div class="report-barrier-questions__question">
                     <h4 class="govuk-heading-s question-label">Export type</h4>
-                    <div class="answer">{% for type in export_type %}{{type|lower|capfirst}} <br> {% endfor %}</div>
+                    <div class="answer">{% for type in export_types %}{{type|lower|capfirst}} <br> {% endfor %}</div>
```
Comment (osimuka) [inline templates/reports/barrier_details_summary_wizard_step.html:95]: do you want to call this something else e.g. `type_` as type is a reserved key word in python?

Comment (chris-pettinga) [reply templates/reports/barrier_details_summary_wizard_step.html]: good point, fixed.

File: barriers/views/export_types.py (hunk 1 above)
Comment (chris-pettinga) [reply barriers/views/export_types.py]: thanks, fixed

**Review by osimuka — APPROVED** [review]

**Pull request merged**